cmd/oras/push.go

```diff
@@ -69,6 +74,9 @@ Example - Push file to the HTTP registry:
 `,
 		Args: cobra.MinimumNArgs(1),
 		PreRunE: func(cmd *cobra.Command, args []string) error {
+			if opts.artifactType != "" && opts.manifestConfigRef != "" {
+				return errors.New("--artifact-type and --manifest-config cannot both be provided")
```

Comment:
probably you can use `fmt.Errorf()` instead.

Reply:
I see in this repo, both errors.New("") and fmt.Errorf("") are used somewhere. However, fmt.Errorf("") is always used along with a formatted string. Since there is no format argument in this case, so maybe we should choose to use errors.New("")?

Reply:
If there is no string interpolation, `errors.New` is preferred.